exercises/diamond/test/Tests.hs

```diff
@@ -3,15 +3,32 @@
 import Data.Foldable     (for_)
 import Test.Hspec        (Spec, describe, it, shouldBe)
 import Test.Hspec.Runner (configFastFail, defaultConfig, hspecWith)
+import Test.QuickCheck   (Gen, elements, forAll, choose)
+import Data.List         ((\\))
+import Data.Maybe        (isNothing, fromMaybe)
+import Data.Char         (ord)
 import Diamond (diamond)
 
 main :: IO ()
 main = hspecWith defaultConfig {configFastFail = True} specs
 
 specs :: Spec
-specs = describe "diamond" $ for_ cases test
-  where
+specs = describe "diamond" $ do
+  it "non-Alpha characters should produce `Nothing`" $
```

Comment:
A mindless omission: The point of the combinator's name, `it`, is to produce a sentence in the output that begins with "diamond ...". I would not demand that we respect this, as the only artifact is in a sentence like "diamond Check the middle of the diamond is correct" when a test fails. Those sentences show up on their own (indented) lines otherwise.

Reply:
I didn't know (or realize) this. I will try to rewrite the descriptions with this in mind.

Reply:
Hspec is inspired by [RSpec](https://rspec.info/about/) which calls itself "behavior-driven development".